Neos.Flow/Classes/Persistence/Doctrine/Repository.php

```diff
@@ -142,51 +136,23 @@ public function findAll(): QueryResultInterface
     /**
      * Find all objects and return an IterableResult
```

Comment:
btw no `IterableResult` here *g

but i was actually wondering .. what is the benefit here of using this vs `findAll`? isnt `findAll` an iterator as well??? I mean we can keep it for b/c ... but i dont know if we need to use it in neos?

Reply:
That _depends_ I guess worth a look at the source, but findAll would return a full array, woudln't it? While this here can be optimized at the driver level to stream the results as single chunks...